### PR TITLE
navigator/mavlink: photos and videos at waypoints

### PR DESCRIPTION
**Pull request by julianoes**
This adds the capability to forward the commands
- MAV_CMD_VIDEO_START_CAPTURE
- MAV_CMD_VIDEO_STOP_CAPTURE
- MAV_CMD_IMAGE_START_CAPTURE
- MAV_CMD_IMAGE_STOP_CAPTURE
at waypoints which will then be sent over mavlink to component id 100 aka MAV_COMP_ID_CAMERA.

@bkueng, @LorenzMeier please review

### CODE REVIEW

File: src/modules/mavlink/mavlink_mission.cpp [hunk 1]
```diff
@@ -981,6 +981,10 @@ MavlinkMissionManager::parse_mavlink_mission_item(const mavlink_mission_item_t *
 		case MAV_CMD_DO_CHANGE_SPEED:
 		case MAV_CMD_DO_SET_SERVO:
 		case MAV_CMD_DO_DIGICAM_CONTROL:
+		case MAV_CMD_IMAGE_START_CAPTURE:
```
Comment (dagar) [inline src/modules/mavlink/mavlink_mission.cpp]: This is really picky, but can you keep the entire thing sorted by value? It makes it easier to keep it consistent in all the different places.

Comment (julianoes) [inline src/modules/mavlink/mavlink_mission.cpp]: Sure, makes sense.

Comment (julianoes) [inline src/modules/mavlink/mavlink_mission.cpp]: Fixed in a8d6276.

**Comment by LorenzMeier** [comment]
Rebased

**Comment by LorenzMeier** [comment]
LGTM

**Pull request merged**